docs/general/ad-filtering/create-own-filters.md

```diff
@@ -518,19 +518,25 @@ Rules with `$match-case` modifier currently are not supported by [AdGuard for iO
 
 #### **`$method`** {#method-modifier}
 
-This modifier limits the rule scope to requests that use the specified set of HTTP methods. Negated methods are allowed. The methods must be specified in all lowercase characters, but are matched case-insensitively. To add multiple methods to one rule, use the vertical bar `|` as a separator.
+This modifier limits the rule scope to requests that use the specified set of [HTTP methods][http-methods].
```

Comment:
CoreLibs do not restrict the available set of methods to ones mentioned on MDN

Reply:
but what other HTTP methods are available for requests?

possible values of $method are restricted in tsurlfilter:
https://github.com/AdguardTeam/tsurlfilter/blob/master/packages/tsurlfilter/src/modifiers/method-modifier.ts#L60
 
shouldn't they be restricted by CoreLibs as well?

Reply:
> but what other HTTP methods are available for requests

for example, standardized http://www.iana.org/assignments/http-methods/http-methods.xhtml

> shouldn't they be restricted by CoreLibs as well

I'm not sure it is really needed. In theory, a method can have an arbitrary value.

Reply:
what if I change the current MDN url to iana.org one?

Reply:
CoreLibs still do not restrict the scope. :) Despite that, there is a sense to add the link but with a note that it is just for reference and the set of available methods is not restricted to them.